src/tray.cpp

```diff
@@ -348,7 +348,8 @@ bool Tray::eventFilter(QObject * object, QEvent * event)
 
 void Tray::onEditConnectionsTriggered()
 {
-    const QStringList connections_editor = QSettings{}.value(CONNECTIONS_EDITOR, QStringList{{"xterm", "-e", "nmtui-edit"}}).toStringList();
+    const QStringList connections_editor = QSettings{}.value(CONNECTIONS_EDITOR, QStringLiteral("xterm -e nmtui-edit")).toString().split(' ');
+
```

Comment:
This is completely unnecessary, as the `QSettings` support storing the string list value, e.g. for your required case:
```
[General]
connectionsEditor=systemsettings,kcm_networkmanagement
```

Reply:
It will be parsed as a list? I didn't know.
However it may be less user friendly, most other places use the space as a delimiter, like desktop files, systemd services, etc.

The parser could be better though, I don't know if one exists in Qt.

Reply:
> his is completely unnecessary, as the `QSettings` support storing the string list value, e.g. for your required case:

Hi, I can confirm using comma works correctly but it feels weird and does not follow usual shell syntax, which is how average user would expect to run manually.



> The parser could be better though, I don't know if one exists in Qt.

As for parsing, actually there is `QProcess::splitCommand()` which takes settings value as single string and returns a string list. This is better because it takes into account quotes so parsing is more correct.

If you want I can make a new PR or you could @j8r could update this one.

Reply:
We may want to know what's the opinion of the maintainers about it to avoid potentially losing time.
What do you think about it @palinek ?
While minor, this is also a breaking change.

Reply:
IMO, this change is pointless and even can break existing user config. More valuable would be to implement GUI for changing this value.